src/app/loadout-builder/types.ts

```diff
@@ -9,6 +9,9 @@ export type StatTypes =
   | 'Discipline'
   | 'Intellect'
   | 'Strength';
+
+export type StatTypesWithTotal = StatTypes | 'Total';
```

Comment:
I was tempted to turn all these string values into enums so I can use them in the other files. I left it as I kept the convention but I think it might be nicer to do this. Let me know if so.